src/panels/history/ha-panel-history.ts

```diff
@@ -174,6 +187,48 @@ class HaPanelHistory extends LitElement {
     this._isLoading = false;
   }
 
+  private _getEntityIds(): string[] {
```

Comment:
I'm sure there's a way better way to do this, but I only use JS/TS for my toy projects so I really don't know how to handle this any better, advice is appreciated!